sample/java/src/main/java/com/datadog/android/sample/MainActivity.java

```diff
@@ -131,11 +134,19 @@ private boolean switchToFragment(@IdRes int id) {
                 spanName = "SwitchingToWebViewFragment";
                 fragmentToUse = WebFragment.newInstance();
                 break;
-            default:
+            case R.id.navigation_traces:
                 mLogger.i("Switching to fragment: Traces");
                 spanName = "SwitchingToTracesFragment";
                 fragmentToUse = TracesFragment.newInstance();
                 break;
+            case R.id.navigation_user:
+                mLogger.i("Switching to fragment: User");
+                spanName = "SwitchingToUserFragment";
+                fragmentToUse = UserFragment.newInstance();
+                break;
+            default:
+                mLogger.e("Switching to unknown fragment " + id);
```

Comment:
there's no 5th tab in that tab menu, why do you add this 4th case here ?

Reply:
Just out of habit and to be future proof. If someone adds another item, it will be caught and logged :)